components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -182,69 +179,70 @@ const ContributorProfileHeader = ({
                 </Button>
 
                 {!isOwner && (
-                  <DropdownMenuTrigger title="More options" className="p-2 mr-3 bg-white rounded-full cursor-pointer ">
+                  <PopoverTrigger title="More options" className="p-2 mr-3 bg-white rounded-full cursor-pointer ">
                     <TfiMoreAlt size={20} className="" />
-                  </DropdownMenuTrigger>
+                  </PopoverTrigger>
                 )}
               </div>
 
-              <DropdownMenuContent align="end" className="flex flex-col gap-1 py-2 rounded-lg">
+              <PopoverContent
+                align="end"
+                className="flex flex-col gap-1 py-2 px-2 !z-40 w-36 text-sm font-medium rounded-lg bg-white outline-none"
+              >
                 {user ? (
                   !isOwner && (
                     <>
-                      <DropdownMenuItem
+                      <button
                         onClick={handleFollowClick}
-                        className="rounded-md flex items-center gap-1 !cursor-pointer [&>span>span:nth-child(1)]:hover:hidden [&>span>span:nth-child(1)]:focus:hidden [&>span>span:nth-child(2)]:hover:inline [&>span>span:nth-child(2)]:focus:inline"
+                        className="rounded-md flex gap-1 !cursor-pointer hover:text-sauced-orange hover:bg-orange-100 [&>span>span:nth-child(1)]:hover:hidden [&>span>span:nth-child(1)]:focus:hidden [&>span>span:nth-child(2)]:hover:inline [&>span>span:nth-child(2)]:focus:inline"
                       >
                         {/* `span` tag changes below must be in line with the styles on the parent */}
                         <span className="pl-3 pr-7">
                           {isFollowing ? (
-                            <>
-                              <span className="">Following</span>
-                              <span className="hidden">Unfollow</span>
-                            </>
+                            <div className="group relative">
```

Comment:
```suggestion
                            <>
```
Keep this as a fragment so the hover effect works on not just the text but the fill width of the button

https://github.com/open-sauced/app/assets/62995161/1c1d8378-4819-48dc-be20-dade8874aea8